src/components/Homepage/components/HomepageImage/HomepageImage.jsx

```diff
@@ -1,12 +1,13 @@
 import React from "react";
 
 import { ImageWrapper, H1 } from "./HomepageImageElements";
-import { ContentWrapper } from "../../../../common/ContentWrapper";
+import SearchBar from "../SearchBar";
 
-export const HomepageImage = () => (
-  <ImageWrapper>
-    <ContentWrapper>
+export const HomepageImage = (props) => {
+  return (
+    <ImageWrapper>
       <H1>Your favourite food from the neighbourhood</H1>
-    </ContentWrapper>
-  </ImageWrapper>
-);
+      <SearchBar handleSearchValue={props.handleSearchValue} />
+    </ImageWrapper>
```

Comment:
I don't think it's correct to call this `ImageWrapper` since it's wrapping `H1` and `SearchBar`. Are they images?

Reply:
Ok, will rename it to Wrapper